src/Components/daterange/Week.svelte

```diff
@@ -0,0 +1,243 @@
+<script>
```

Comment:
I do believe this could be mixed into https://github.com/6eDesign/svelte-calendar/blob/master/src/Components/Week.svelte itself, and just base it on if selectedEnd is set or not.